src/interpreter.js

```diff
@@ -173,15 +173,19 @@ function created(utcSeconds) {
 }
 
 function tsCreated(ts) {
-  var date = new Date(ts)
-  var dateString =
-    MONTHS[date.getUTCMonth()] + ' ' +
-    padLeft(date.getUTCDate().toString(), 2, ' ') + ' ' +
-    padLeft(date.getUTCHours().toString(), 2, '0') + ':' +
-    padLeft(date.getUTCMinutes().toString(), 2, '0') + ':'  +
-    padLeft(date.getUTCSeconds().toString(), 2, '0')
-
-  return `<span class="created">${padSpacesRight((date.getTime() / 1000).toString(), 14)} </span><span class="date">${dateString} </span>`
+  if (typeof(ts) == "string") {
+    var date = new Date(ts)
+    var dateString =
+      MONTHS[date.getUTCMonth()] + ' ' +
+      padLeft(date.getUTCDate().toString(), 2, ' ') + ' ' +
+      padLeft(date.getUTCHours().toString(), 2, '0') + ':' +
+      padLeft(date.getUTCMinutes().toString(), 2, '0') + ':'  +
+      padLeft(date.getUTCSeconds().toString(), 2, '0')
+
+    return `<span class="created">${padSpacesRight((date.getTime() / 1000).toString(), 14)} </span><span class="date">${dateString} </span>`
+  } else {
+    return created(ts)
```

Comment:
not sure what/why we have two almost identical functions, but I guess that's out of the scope of this fix.
Perhaps a little nicer it so have single `created` function that handles both cases (number and string)?

Reply:
extracted the common code to a function